examples/multi-project/README.md

```diff
@@ -115,11 +117,14 @@ jobs:
           infracost output --path="infracost_jsons/*.json" --format=json --out-file=/tmp/infracost_combined.json
```

Comment:
We don't need `output` command anymore. The `infracost comment` can accept the blob pattern. The whole step can be removed.

Reply:
👍

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2

Reply:
I'm getting test failures with this though: https://dev.azure.com/infracost/base/_build/results?buildId=1056&view=logs&j=4153937c-ab17-5c86-5eb6-7b0ab0c53920&t=a66737fe-e920-5002-ef61-aac67ed1ea1e

Reply:
is this expected, it looks like it's not picking up all the project outputs in `infracost comment`